Services/UI/OJS.Services.Ui.Business/Validations/Implementations/Contests/ContestValidationService.cs

```diff
@@ -67,7 +67,7 @@ public ValidationResult GetValidationResult((Contest?, int?, string, bool, bool)
         {
             if (participant.ParticipationEndTime != null)
             {
-                return DateTime.Now >= participant.ParticipationEndTime;
+                return DateTime.Now >= participant.ParticipationEndTime.Value;
```

Comment:
Shouldn't this also be `DateTime.UtcNow`?

Reply:
At the current moment, we have the problem of adding +3 hours after refresh, that's why I left it for DateTime.Now.